src/org/sosy_lab/java_smt/solvers/z3/Z3AbstractProver.java

```diff
@@ -73,6 +74,34 @@ abstract class Z3AbstractProver<T> extends AbstractProverWithAllSat<T> {
 
     logfile = pLogfile;
     mgr = pMgr;
+    postProcessProverCreation(pSolverOptions);
+  }
+
+  Z3AbstractProver(
```

Comment:
This looks like redundant code in both constructors..

Reply:
How so?